plugin/PluginTidyModule.cpp

```diff
@@ -28,12 +28,12 @@ class PluginModule : public ClangTidyModule {
 
 } // namespace plugin
 
-// Register the AliceO2TidyModule using this statically initialized variable.
+// Register the PluginO2TidyModule using this statically initialized variable.
 static ClangTidyModuleRegistry::Add<plugin::PluginModule>
-X("aliceO2-module", "Adds AliceO2 specific checks");
+X("pluginO2-module", "Adds PluginO2 specific checks");
```

Comment:
changes like here are not related to this PR. It is preferred to have them in a separate commit/PR to better understand development history.